prov/verbs/src/verbs_info.c

```diff
@@ -1117,35 +1103,44 @@ static int fi_ibv_set_default_info(struct fi_info *info)
 	int ret;
 
 	ret = fi_ibv_set_default_attr(info, &info->tx_attr->size,
-				      verbs_default_tx_size, "tx context size");
+				      fi_ibv_gl_data.def_tx_size,
+				      "tx context size");
 	if (ret)
 		return ret;
 
 	ret = fi_ibv_set_default_attr(info, &info->rx_attr->size,
-				    verbs_default_rx_size, "rx context size");
+				      fi_ibv_gl_data.def_rx_size,
+				      "rx context size");
 	if (ret)
 		return ret;
 
-	/* Don't set defaults for verb/RDM as it supports an iov limit of just 1 */
+	/* Don't set defaults for verb/RDM as
+	 * it supports an iov limit of just 1 */
 	if (info->ep_attr->type != FI_EP_RDM) {
-		ret = fi_ibv_set_default_attr(info, &info->tx_attr->iov_limit,
-					      verbs_default_tx_iov_limit,
-					      "tx iov_limit");
+		ret = fi_ibv_set_default_attr(
+			info, &info->tx_attr->iov_limit,
+			fi_ibv_gl_data.def_tx_iov_limit,
+			"tx iov_limit");
 		if (ret)
 			return ret;
 
-		/* For verbs iov limit is same for both regular messages and RMA */
-		ret = fi_ibv_set_default_attr(info, &info->tx_attr->rma_iov_limit,
-					      verbs_default_tx_iov_limit,
-					      "tx rma_iov_limit");
+		ret = fi_ibv_set_default_attr(
+			info, &info->rx_attr->iov_limit,
+			fi_ibv_gl_data.def_rx_iov_limit,
+			"rx iov_limit");
 		if (ret)
 			return ret;
 
-		ret = fi_ibv_set_default_attr(info, &info->rx_attr->iov_limit,
-					      verbs_default_rx_iov_limit,
-					      "rx iov_limit");
-		if (ret)
-			return ret;
+		if (info->ep_attr->type != FI_EP_DGRAM) {
+			/* For verbs iov limit is same for
+			 * both regular messages and RMA */
+			ret = fi_ibv_set_default_attr(
+				info, &info->tx_attr->rma_iov_limit,
+				fi_ibv_gl_data.def_tx_iov_limit,
+				"tx rma_iov_limit");
```

Comment:
nit: The indentation style for the function arguments is not consistent with rest of the libfabric code. The indent should align with one space after open parentheses like in the code that's being deleted.